Orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -1304,7 +1378,12 @@ Function Invoke-Bootstrap {
             throw "ToolkitComponents.Audit.StorageType not supported, currently supported types are: StorageAccount and Local";
         }
 
-        return $factory;
+        # Do not return the factory object. Instead do a Write-Output of the factory object.
```

Comment:
We haven't ran into this issue? Can you explain why returning factory breaks the execution? If two items are returned, maybe you can return a hashtable?

Reply:
There are two approaches to this. Returning a single object that will wrapped all the other objects we would like to return. Second option - using something similar to destructuring  assignment.

For the first option:
* Pro
- Use return statement consistent with usage in other places in our code
* Cons
- Additional changes to the accommodate the wrapped objects. In our case, factory is no longer the global var, it is a sub-property of another parent global var.

For the second option:
* Pros:
- Less changes to existing vars
* Cons:
- New usage of Write-Output for return instead of return statement

Reply:
As per conversation, converted this to use first option above.